workspaces/repo-tools/README.md

```diff
@@ -1,3 +1,41 @@
 # Community Plugins Repo Tooling
 
-This repository contains the tools and scripts used to manage the community plugins repository.
+CLI tools for managing the community plugins repository.
+
+## Usage
+
+```bash
+npx community-cli <command>
```

Comment:
isn't safer to use yarn everywhere instead of npx? Thinking about if someone accidentally misspell `community-cli`

```suggestion
yarn community-cli <command>
```

Reply:
sorry for laziness I suggested to only change the reference above 😅 shall we change all the other occurrences of `npx` as well?